tests/pools/cluster/fixed.test.js

```diff
@@ -87,7 +98,7 @@ describe('Fixed cluster pool test suite', () => {
       inError = e
     }
     expect(inError).toBeDefined()
-    expect(typeof inError === 'string').toBeTruthy()
+    expect(typeof inError === 'string').toEqual(true)
```

Comment:
Why not `.toBeTruthy()`? I think we can safely use it here

Reply:
Stricter returned value check: typeof return a boolean, never null, '', etc.